docs/deploying-phero/gcloud-functions.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Deploying to GCloud Functions (a.k.a. Firebase Functions)"
+---
+
+Google Cloud offers serverless functions. Google calls these `Google Cloud Functions`. Some people know them as `Firebase Functions`. Under the hood they are the same thing.
+
+In order to deploy your Phero server, there's one last step: making an export.
+When you export you server, Phero compiles an optimized server for you to deploy.
+
+## Prerequisites
+
+Make sure you've installed `gcloud CLI`. If you haven't installed it yet, follow the steps here: https://cloud.google.com/sdk/docs/install
+
+## Steps
+
+First navigate to your server dir:
+
+```
+cd api
+```
+
+```
+npm install @google-cloud/functions-framework
+```
+
+To make an export of your Phero services, run:
+
+```
+npx phero@latest server export --flavor gcloud-functions
+```
+
+This will output an `export` directory with a directory for each `service`.
```

Comment:
Didn't we think defining the export-directory in the command (like `npx phero server export ./export --flavor gcloud-functions`) was a bit more flexible across the different flavors? Or is that something to fix later on?